### ISSUE TITLE
Top-down map should be a connected graph

### ISSUE THREAD

**Issue by pengzhenghao** [decisionforce/pgdrive#184]
When running:

(pseudo code)
```
import matplotlib.pyplot as plt
plt.imshow(self.get_map(resolution=(512, 512)))
plt.show()
```

Resolution: (512, 512):
<img width="402" alt="image" src="https://user-images.githubusercontent.com/22206995/103174623-9cf0ee00-489e-11eb-9dae-19b121fcef41.png">

Resolution: (64, 64):
<img width="372" alt="image" src="https://user-images.githubusercontent.com/22206995/103174610-88145a80-489e-11eb-922d-6913e97e2811.png">

You can find that the low-resolution image is not connected. I think the down-sampling operation should not influence the connectivity of the top-down image.

**Comment by pengzhenghao** [comment]
Please solve this in a new PR @lqy0057 , then I will finish the basic framework in #185

**Comment by lqy0057** [comment]
I reproduce this, and will change the way to get map image

**Comment by lqy0057** [comment]
The pygame.transform.scale() can not solve this, so I am  working on optimizing our pooling() function now

**Comment by lqy0057** [comment]
The time consuming of pooling by doing pooling on numpy.ndarray: 129s
![image](https://user-images.githubusercontent.com/24489294/103199736-6a8ad380-4926-11eb-9f06-30347169dc9a.png)
The time consuming of pooling by getting pixel value on the surface:27.966s
![image](https://user-images.githubusercontent.com/24489294/103199779-9312cd80-4926-11eb-8fbd-03179912fe33.png)

**Comment by pengzhenghao** [comment]
Link #193 , wait for @lqy0057 review.

**Comment by lqy0057** [comment]
fixed in #193
